packages/components/affix/src/affix.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div ref="root" :class="ns.b()" :style="rootStyle">
-    <div :class="{ [ns.m('fixed')]: fixed }" :style="affixStyle">
-      <slot />
-    </div>
+    <el-teleport :disabled="teleportDisabled" :to="props.appendTo">
+      <div :class="{ [ns.m('fixed')]: fixed }" :style="affixStyle">
+        <slot />
+      </div>
+    </ElTeleport>
```

Comment:
```suggestion
    </el-teleport>
```

Sorry, I forgot to submit this part.